plugins/rikki/heroeslounge/models/SlothTeamPivot.php

```diff
@@ -0,0 +1,51 @@
+<?php namespace Rikki\Heroeslounge\Models;
+
+use October\Rain\Database\Pivot;
+use Rikki\Heroeslounge\Models\Sloth;
+use Rikki\Heroeslounge\Models\Team;
+
+/**
+ * Sloth-Team Pivot Model
+ */
+class SlothTeamPivot extends Pivot
+{
+    
+    /*
+     * Disable timestamps by default.
+     * Remove this line if timestamps are defined in the database table.
+     */
+    public $timestamps = false;
+
+    /**
+     * @var string The database table used by the model.
+     */
+    public $table = 'rikki_heroeslounge_sloth_team';
+
+    public function afterSave()
+    {
+        if ($this->isDirty('is_captain')) {
+            $sloth = Sloth::find($this->sloth_id);            
+
+            if ($this->is_captain) {
+                $sloth->addDiscordCaptainRole();
```

Comment:
This causes unnecessary calls to the Discord API but this should be ok

Reply:
Yeah I debated doing a check if the user was already a captain, but I also remembered that we've done captain purging in the past, so the website is not always 100% in sync with Discord in that regard...